app/models/user_orderable_class.rb

```diff
@@ -0,0 +1,13 @@
+module UserOrderableClass
```

Comment:
I'd probably call this one `UserOrderable` and the extension `UserOrderableExtension` or `UserOrderableRelation` I think. Calling a module a class is potentially confusing!

I'd actually consider nesting the extension inside this module, but not too sure what the impact of that would be.

Reply:
cool have done. I'll give nesting the extension a go